elkm1_lib/keypads.py

```diff
@@ -30,13 +44,17 @@ def __init__(self, connection: Connection, notifier: Notifier) -> None:
         notifier.attach("IC", self._ic_handler)
         notifier.attach("KA", self._ka_handler)
         notifier.attach("KC", self._kc_handler)
+        notifier.attach("KF", self._kf_handler)
         notifier.attach("LW", self._lw_handler)
         notifier.attach("ST", self._st_handler)
 
     def sync(self) -> None:
         """Retrieve areas from ElkM1"""
         self._connection.send(ka_encode())
         self.get_descriptions(TextDescriptions.KEYPAD.value)
+        # Not sure about this, sending kf needs a keypad index, so should we send all?
+        # Just send one for now
```

Comment:
Yup, send for each keypad.

Reply:
Committed untested fix, will test tonight.